README.md

```diff
@@ -1471,6 +1490,23 @@ Listener used to receive log messages from the SDK.
 ### Enums
 
 
+#### PURCHASES_ARE_COMPLETED_BY
+
+| Members          | Value                     | Description                                                                                                                                                                                                                                                                                                                                |
+| ---------------- | ------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **`MY_APP`**     | <code>"MY_APP"</code>     | RevenueCat will **not** automatically acknowledge any purchases. You will have to do so manually. **Note:** failing to acknowledge a purchase within 3 days will lead to Google Play automatically issuing a refund to the user. For more info, see [revenuecat.com](https://docs.revenuecat.com/docs/observer-mode#option-2-client-side). |
```

Comment:
It's worth mentioning here that if you use `MY_APP`, you also need to let us know which StoreKit version you're using so that we can use the correct implementation.

Reply:
Hmm this actually comes from PHC, so we should add that there.

Reply:
This has actually changed in PHC 13.0.0